pkg/cmd/repo/license/view/view.go

```diff
@@ -0,0 +1,99 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	License    string
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Command {
+	opts := &ViewOptions{
+		IO:         f.IOStreams,
+		HTTPClient: f.HttpClient,
+		Config:     f.Config,
+		License:    "",
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <license>",
+		Short: "View an available repository license template",
+		Long: heredoc.Docf(`
+		View an available repository license template.
+		
+		%[1]s<license>%[1]s is a license name or SPDX ID.
+		`, "`"),
+		Args: cmdutil.ExactArgs(1, "gh repo licese view only takes a single license argument"),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.License = args[0]
+			if runF != nil {
+				return runF(opts)
+			}
+			return viewRun(opts)
+		},
+	}
+	return cmd
+}
+
+func viewRun(opts *ViewOptions) error {
+	if opts.License == "" {
+		return errors.New("no license provided")
+	}
+
+	client, err := opts.HTTPClient()
+	if err != nil {
+		return err
+	}
+
+	cfg, err := opts.Config()
+	if err != nil {
+		return err
+	}
+
+	if err := opts.IO.StartPager(); err == nil {
+		defer opts.IO.StopPager()
+	} else {
+		fmt.Fprintf(opts.IO.ErrOut, "failed to start pager: %v\n", err)
+	}
```

Comment:
Technically works but more idiomatic and consistent if reswizzled like [`gh gist view`](https://github.com/cli/cli/blob/dfddb16ba34d5e7b2fb912c7fb6426c5cd2951c5/pkg/cmd/gist/view/view.go#L128-L131):

```suggestion
	if err := opts.IO.StartPager(); err != nil {
		fmt.Fprintf(opts.IO.ErrOut, "starting pager failed: %v\n", err)
	}
	defer opts.IO.StopPager()
```